### PR TITLE
Update from Weblate

### PR DESCRIPTION
**Pull request by xwikiorgci**

### CODE REVIEW

File: src/main/resources/Help/Code/Translations.fr.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+help.templates.examples.description=Vous voulez commencer à partir d'un modèle prédéfini ? Voici quelques exemples de modèles disponibles dans XWiki
+help.templates.examples.title=Exemples
+help.templates.meeting.title=Compte-rendu de réunion
+help.templates.meeting.description=Agenda et notes
+help.templates.simple.description=Avec table des matières
+help.templates.simple.title=Page simple
+help.templates.more=Voir plus de modèles
+help.templates.title=Modèles
+help.templates.useTemplates.description=Les modèles sont des pages prédéfinies qui sont copiées lors de la création d'une nouvelle page.
+help.templates.useTemplates.tip={0}Créer des pages{1}à partir de modèles permet d''adapter rapidement un contenu prédéfini à vos besoins.
```
Comment (surli) [inline src/main/resources/Help/Code/Translations.fr.xml]: Looks like there's a missing space?

File: src/main/resources/Help/Code/Translations.fr.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+help.videos.videoCard2.topic1=Apprenez comment modifier le thème de couleur de votre wiki
+help.videos.videoCard2.topic2=Apprenez à ajouter des panels et des colonnes sur votre wiki
+help.videos.videoCard2.topic3=Apprenez comment ajouter des gadgets à votre wiki
+help.videos.videoCard3.title=Création de page et de contenu
+help.videos.videoCard3.topic1=Découvrez comment créer une page
+help.videos.videoCard3.topic2=Apprenez comment utiliser l'éditeur de texte WYSIWYG
+help.videos.videoCard3.topic3=Découvrez les modes vue et édition
+help.videos.videoCard4.title=Importer des documents Office dans XWiki
+help.videos.videoCard4.topic1=Découvrez comment importer des documents Office dans votre wiki
+help.videos.videoCard4.topic2=Apprenez à utiliser la macro "Table des Matières"'
```
Comment (surli) [inline src/main/resources/Help/Code/Translations.fr.xml]: A trailing quote ' to remove at the end?

File: src/main/resources/Help/Code/Translations.fr.xml (hunk 1 above)
Comment (CamiAndrei) [reply src/main/resources/Help/Code/Translations.fr.xml]: I have added the missing space.

File: src/main/resources/Help/Code/Translations.fr.xml (hunk 2 above)
Comment (CamiAndrei) [reply src/main/resources/Help/Code/Translations.fr.xml]: I have removed the ' at the end.

**Pull request merged**